### DISCUSSION TITLE
Guests being able to search pages

### DISCUSSION THREAD

**Discussion by andrewschmidgit** [requarks/wiki#3579] [Error / Bug Report]
**Question**
Unless I have setup something incorrectly, I do not believe Guests are able to search.

Right now, the **Guests** group has these page rules:
| Roles | Operator | Path |
|---|---|---|
| `read:pages`|Tag Matches|`public`|
| `read:assets`|Path Starts With| `assets/public`|

Right now for testing, the home page has the `public` tag

When I am logged in, I am able to search normally

When I am not logged in (ie, Guest), searching anything by the page title, description, or path yields no results.

What did I do wrong?

**Host Info (please complete the following information):**
OS: Alpine 1.11.6 x64 (on Heroku, using the Heroku deploy template)
Wiki.js version: 2.4.107
Database engine: postgres 12.3

**Comment by NGPixel** [comment]
Tags are not part of search data and therefor cannot be used to filter results.

**Comment by despens** [comment]
Same as #1729
